nml/expression/functioncall.py

```diff
@@ -457,25 +457,22 @@ def builtin_str2number(name, args, pos):
 
 
 @builtins("cargotype", "railtype", "roadtype", "tramtype")
-def builtin_resolve_typelabel(name, args, pos):
+def builtin_resolve_typelabel(name, args, pos, table_name=None):
     """
     {cargo,rail,road,tram}type(label) builtin functions.
 
     Also used from some Action2Var variables to resolve cargo labels.
     """
     tracktype_funcs = {
+        "cargotype": global_constants.cargo_numbers,
```

Comment:
Using `cargotype` here will change error messages "...in your cargo table." to "...in your cargotype table."

Given that the syntax is `cargotable {}` vs `railtypetable {}` (etc.) this might not be a desired change.